EPRN.PRNS.API/Repositories/Repository.cs

```diff
@@ -267,5 +281,14 @@ await _prnContext.PRN
                     sp.SetProperty(wj => wj.DecemberWaste, decemberWaste)
                 );
         }
+
+        public async Task SaveSentTo(int id, string status)
+        {
+            await _prnContext.PRN
+                .Where(wj => wj.Id == id)
+                .ExecuteUpdateAsync(sp =>
+                    sp.SetProperty(wj => wj.SentTo, status)
```

Comment:
The parameter name doesn't seem to make sense - status? For the SentTo field?

Reply:
Done.